src/accordion.test.keyboard.ts

```diff
@@ -4,15 +4,17 @@ import { expect, test } from './playwright/test.js';
 test(
   'can be opened via Enter',
   { tag: '@keyboard' },
-  async ({ browserName, mount, page }) => {
-    // eslint-disable-next-line playwright/no-skipped-test
+  async ({ mount, page }) => {
     test.skip(
-      browserName === 'webkit',
+      ({ browserName }) => browserName === 'webkit',
```

Comment:
No need to bring `browserName` into scope for the whole test. Didn't realize it also gets passed to a conditional `test.skip()`.